src/WebUI/src/layouts/default.vue

```diff
@@ -62,7 +62,7 @@ onBeforeUnmount(() => {
 });
 
 await Promise.all(promises);
-await userStore.getUserClanMember(); // TODO: get the clan role in the query `users/self/clans`
+await userStore.getUserClanMember();
```

Comment:
the meaning of this task is to remove this method (delete, also from auto tests) that is.

earlier, to find out the role in the clan, we received a list of all clan members, ok?

https://github.com/verdie-g/crpg/blob/fc4a5b392d0e23e437b821037442af6cc4e6d3f5/src/WebUI/src/stores/user.ts#L73

Reply:
ah, got it, will look into it again